RandomError1.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import division
+__author__ = 'PaulComeau'
+import random
+import argparse
+from sys import argv
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter, FileType
+
+import doctest
+
+newstring=""
+FastQ=open('ABL1_E_r_001_116_R_0_GSP2_R2.fastq', 'r')
```

Comment:
Avoid hard coding file names, instead use argparse to take a fastq file from the command line.